xtask/src/themelint.rs

```diff
@@ -0,0 +1,172 @@
+use crate::path;
+use crate::DynError;
+use helix_view::theme::Modifier;
+use helix_view::Theme;
+
+struct Rule {
+    fg: Option<String>,
+    bg: Option<String>,
+    check_both: bool,
+}
+
+// Placed in an fn here, so it's the first thing you see
+fn get_rules() -> Vec<Rule> {
+    vec![
+        // Check for ui.selection, which is required
+        Rule::has_either("ui.selection".into()),
```

Comment:
I believe we can just do `&'static str` instead of `String` given that all of these we have them when we compile?